Sparks_Steven/Assignments/stars2.py

```diff
@@ -0,0 +1,9 @@
+def drawing_stars2(draw):
+	for item in draw:
+		if type(item) != str:
+			print '*' * item
+		else: 
+			print str.lower (item[0])* len(item) 
```

Comment:
_Never_ put a space between a function and its invocation: `print str.lower(item[0])* len(item)`